subsys/net/ip/ipv4.c

```diff
@@ -143,6 +143,11 @@ static inline enum net_verdict process_icmpv4_pkt(struct net_pkt *pkt,
 
 	icmp_hdr = net_icmpv4_get_hdr(pkt, &hdr);
 
```

Comment:
Usually we do not have empty line after setting the var and then checking it, so you could remove this extra line here.

Reply:
Thanks for info. Updated patch pushed.